### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by mariomui**
makes sure that location/rooms/1 ro location/rooms/2 gets served the entrypoint page.

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
 // const DIRECT_ROUTE = path.join(__dirname, 'public/');
 app.use(express.static('public'));
 
+app.get('/rooms/:id/pictures', function (req, res) {
+  console.log('hello');
+  findPicturesById(req.params.id)
+    .then(function (images) {
+      res.send(images.image_url);
+    });
+});
```
Comment (wusihong) [inline server/index.js]: how do you resolve potential errors from the get request?

File: server/index.js [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
 // const path = require('path');
 
 const { urlencoded, json } = bodyParser;
 const app = express();
 
+app.use(morgan('dev'));
 app.use(urlencoded({ extended: true }));
 app.use(json());
 
 // const DIRECT_ROUTE = path.join(__dirname, 'public/');
```
Comment (wusihong) [inline server/index.js]: general comment on this file:  make sure to clear out unnecessary console logs such as console.log('hello') etc.

File: server/index.js [hunk 3]
```diff
@@ -1,16 +1,27 @@
 const express = require('express');
+const morgan = require('morgan');
+
 const bodyParser = require('body-parser');
+const { findPicturesById } = require('../database/');
 // const path = require('path');
 
 const { urlencoded, json } = bodyParser;
```
Comment (wusihong) [inline server/index.js]: minor point: just consider organizing the lines to be grouped and lumped in a logical way. for example, line 4 is bodyParser related, then you have a line requiring database, then a space, and then back to bodyParser...  helps readability of the flow makes sense and similar things are grouped together. t

File: server/index.js (hunk 1 above)
Comment (mariomui) [reply server/index.js]: i manually delete them.

File: client/components/Landing.jsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-useless-constructor */
+/* eslint-disable class-methods-use-this */
+import React from 'react';
+
+class Landing extends React.Component {
```
Comment (wusihong) [inline client/components/Landing.jsx]: Not sure if Landing accurately describes what this component "is". When I think of landing, i think of a full landing page. Perhaps Carousel...or simply the generic "App" could be more accurate

Comment (mariomui) [reply client/components/Landing.jsx]: I will take that under consideration.

**Pull request merged**